control-plane/api-gateway/translation/config_entry_translation.go

```diff
@@ -20,6 +20,7 @@ const (
 	metaValueManagedBy     = "consul-k8s-gateway-controller"
 	metaKeyKubeNS          = "k8s-namespace"
 	metaKeyKubeServiceName = "k8s-service-name"
+	metaKeyKubeName        = "k8s-name"
```

Comment:
this came from the POC implementation https://github.com/hashicorp/consul-k8s/blob/408114a1fd7c82f2760839a2f22cc77c840208cb/control-plane/controller/apigateway_controller.go#L416, I'm not 100% on which we use as `metaKeyKubeService` name elsewhere in this repo (or do we use both keys in the meta?)

Reply:
I think we can keep it around and if we get to the end of the implementation and find that we aren't using it, we can toss it.